src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -382,15 +426,19 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		}
 	}
 
-	private createButtonPlugin(col: any) {
+	private createButtonPlugin(col: azdata.ButtonColumn) {
```

Comment:
Seems kind of weird to call these `createXPlugin` when they aren't really creating a plugin - just a column

Reply:
these are actually plugins as well. :)

![image](https://user-images.githubusercontent.com/13777222/102151509-ed406880-3e27-11eb-83d5-e869e0b49f03.png)